tools/clang/unittests/HLSL/DXIsenseTest.cpp

```diff
@@ -237,6 +238,16 @@ TEST_F(DXIntellisenseTest, InclusionWhenMissingThenError) {
   VERIFY_SUCCEEDED(pDiag->GetSeverity(&Severity));
   VERIFY_IS_TRUE(Severity == DxcDiagnosticSeverity::DxcDiagnostic_Error ||
     Severity == DxcDiagnosticSeverity::DxcDiagnostic_Fatal);
+  VERIFY_SUCCEEDED(pDiag->GetLocation(&location));
```

Comment:
We should also have a test verifying the failure if the parameter is null.

Reply:
Done